src/rules/comment-word-blacklist/__tests__/index.js

```diff
@@ -6,6 +6,30 @@ const rule = rules[ruleName]
 
 testRule(rule, {
   ruleName,
+  config: ["bad-word"],
+
+  accept: [ {
+    code: "/* comment */",
+  }, {
+    code: "/*! bad-word */",
+  }, {
+    code: "/*# bad-word */",
+  }, {
+    code: "/** bad-word **/",
+  }, {
+    code: "/*** bad-word ***/",
+  } ],
```

Comment:
@evilebottnawi Thoughts?

I think we originally decided [to ignore](https://github.com/stylelint/stylelint/blob/master/src/rules/comment-word-blacklist/index.js#L29) `/*!` and `/*#` as they are used for copyright and sourcemaps respectively, and likely (although the saying the former is is a bit of stretch) out of the users control. 

Not sure why the `/**` is ignored though. 

In `v7` shall we fix this and only ignore sourcemap comments?

Shall I open a separate issue and get this merged?